footer.php

```diff
@@ -24,11 +24,14 @@
 			<div class="site-name">
 				<?php
 				$blog_info = get_bloginfo( 'name' );
-				if ( ! empty( $blog_info ) ) {
-					?>
-					<a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a>
-					<?php
-				}
+				if ( ! empty( $blog_info ) ) :
+					if ( is_front_page() || ( is_front_page() && is_home() ) ) :
+						?>
+						<?php bloginfo( 'name' ); ?>
+					<?php else : ?>
+						<a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"<?php echo $aria_current; ?>><?php bloginfo( 'name' ); ?></a>
```

Comment:
Looks like `$aria_current` needs to be escaped.

Reply:
yep!

Reply:
ah wait, we don't need this $aria_current here